brats/data/datasets.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+from torch.utils import data
+import nibabel as nib
+
+from torchvision.datasets import ImageFolder
+
+
+class NiftiFolder(data.Dataset):
+    """
+    A custom loader for .nii.gz files in a single folder.
+    Each file should contain a scan of a single patient in one or more modalities.
+    E.g.:
+    scans/patient000.nii.gz
+    scans/patient001.nii.gz
+    scans/patient002.nii.gz
+    where file scans/patient000.nii.gz contains scan of the patient 001 in 4 modalities:
+    T1, T1gd, T2w, Flair
```

Comment:
It appears that files have modalities in the following order:
0: FLAIR
1: T1w
2: T1gd
3: T2w

Maybe we should include this order in that docstring?

Reply:
I don't want to suggest that this is the proper order (I believe pure brats data have different one). But I agree that this should be noted somewhere. 

Maybe I could add a note in this docstring concerning the fact that order is not important, and add to wiki a description of the dataset?

Reply:
Sure, good idea.